critiquebrainz/frontend/external/musicbrainz.py

```diff
@@ -73,6 +73,27 @@ def browse_release_groups(artist_id=None, release_types=None, limit=None, offset
     return release_groups
 
 
+def browse_releases(artist_id=None, release_types=None, limit=None, offset=None, release_group=None, includes=None):
```

Comment:
I'd move `release_group` to the beginning after `artist_id` since it can be considered a "main" filtering parameter.